ajax/libs/videojs-hotkeys/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "filename": "videojs.hotkeys.min.js",
+  "name": "videojs-hotkeys",
+  "version": "0.2.18",
+  "description": "Adds more hotkey support to video.js",
+  "author": "Chris Dougherty",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ctd1500/videojs-hotkeys.git"
+  },
+  "keywords": [
+    "videojs",
+    "videojs-plugin",
+    "hotkeys"
+  ],
```

Comment:
Would you add `"videojs"` in keywords?
I found it show here https://github.com/ctd1500/videojs-hotkeys/blob/master/bower.json#L15

Add more keyword can let people search the library more easier.